.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: python
 
+notifications:
+  email: false
+
```

Comment:
That probably shouldn't go to production

Reply:
@marioba can you undo this change?

Reply:
@m-kuhn done!